Sources/Extensions/UIKit/UISearchBarExtensions.swift

```diff
@@ -37,5 +28,16 @@ public extension UISearchBar {
 	public func clear() {
 		text = ""
 	}
+    
+    /// SwifterSwift: Text field inside search bar (if applicable).
+    ///
+    /// - Returns: The instance for embedded textField.
+    public func textField() -> UITextField? {
+        let subViews = subviews.flatMap { $0.subviews }
+        guard let textField = (subViews.filter { $0 is UITextField }).first as? UITextField else {
```

Comment:
Nasty bit of code 🤔 This is probably a good place to use `first(where:(Element)->Bool)`